pushnotifications/src/main/java/com/pusher/pushnotifications/api/DeviceHeadersInterceptor.kt

```diff
@@ -0,0 +1,18 @@
+package com.pusher.pushnotifications.api
+
+import com.pusher.pushnotifications.BuildConfig
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+
+class DeviceHeadersInterceptor : Interceptor {
```

Comment:
I think this could have a more meaningful name, e.g. `PusherLibraryHeaderInterceptor`?